pkg/errtypes/errtypes.go

```diff
@@ -187,3 +191,33 @@ type IsChecksumMismatch interface {
 type IsInsufficientStorage interface {
 	IsInsufficientStorage()
 }
+
+// NewErrtypeFromStatus maps an rpc status to an errtype
+func NewErrtypeFromStatus(status *rpc.Status) error {
```

Comment:
this is also used only in one place. For this to be widely adopted I'm missing some guidelines or docs that refer to `errtypes.NewErrtypeFromStatus`. It can be easily done in a followup PR. But it is true that the scopes of these changes are too wide, and these idioms fall between the cracks 👍

Reply:
Would you prefer to update documentation or to remove the function?